### PR TITLE
rocAL - API Updates

### PR DESCRIPTION
**Pull request by rrawther**
This PR to fix rocAL video unit tests

### CODE REVIEW

**Review by paveltc — APPROVED** [review]

**Comment by paveltc** [comment]
@kiritigowda I ran tests on this PR and it can be merged.

File: rocAL/rocAL/source/api/rocal_api_data_loaders.cpp [hunk 1]
```diff
@@ -2139,7 +2140,12 @@ rocalVideoFileResize(
         bool loop,
         unsigned step,
         unsigned stride,
-        bool file_list_frame_num)
+        bool file_list_frame_num,
+        RocalResizeScalingMode scaling_mode,
+        std::vector<unsigned> max_size,
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/api/rocal_api_data_loaders.cpp:2145]: @rrawther Seems like there is a codacy warning

**Comment by kiritigowda** [comment]
@LakshmiKumar23 can you verify with RPP V0.99

**Comment by kiritigowda** [comment]
@LakshmiKumar23 let me know if this is ready to be merged

**Comment by paveltc** [comment]
@kiritigowda @rrawther It seems all of the changed files have disappeared from this PR.

**Comment by LakshmiKumar23** [comment]
@kiritigowda this PR can be closed. @shobana-mcw will create a new PR in it's place

**Comment by rrawther** [comment]
@shobana-mcw will create a replacement PR for this

**Pull request closed without merging**